### ISSUE TITLE
Extension does not detect adds

### ISSUE THREAD

**Issue by denbell5** [denbell5/TfsUndoAndDeleteNewFilesExtension#1]
Sometimes extension does not detect adds and prints the following to the output pane:
```
Executing UndoAndDeleteNewFilesCommand [03/06/2023 12:47:05 PM]
Found 0 'adds' before undo
Executing TeamFoundationContextMenus.PendingChangesPageChangestoInclude.TfsContextPendingChangesPageUndo
Execution succeeded
Found 0 'adds' after undo
Found 0 undone 'adds'
```
I do not know yet exact steps to reproduce, the issue is not always replicable.
Restart of Visual Studio fixes the issue.

**Comment by denbell5** [comment]
[Commited](https://github.com/denbell5/TfsUndoAndDeleteNewFilesExtension/commit/5750466114bfa2a926e856bd4117c8a1f05cea79)
the change to not proceed with execution if this case detected.

**Comment by denbell5** [comment]
Fixed by #4
